wrappers/kratosWrapper.py

```diff
@@ -51,68 +47,127 @@ def __init__(self):
         self.SP = DataContainer()
         self.BC = DataContainer()
 
+        # Extended data containers
+        self.SPE = {}
+
         # Initialization
         self.initialize()
 
-    def __addNodalVariablesToModelpart(self):
-        pass
+    # ABCModelingEngine Implementation
+
+    def add_particles(self, particle_container):
+        message = 'KratosWrapper does not handle particle container'
+        raise NotImplementedError(message)
+
+    def add_mesh(self, src):
```

Comment:
This deviates from the ABCModelingEngine as:
1) it does not return anything
2) just keeps the original 'src' mesh.  

For example, (2) would be a problem:
```
 with closing(H5CUDS.open("foo.cuds", 'r')) as file_handle:   
        wrapper.add_mesh(file_handle.get_mesh("meshA"))
        wrapper.run()
```
something along the lines of this should be done:
```
def add_mesh(self, src):
    copy_mesh = Mesh(name=src.name)
    copy_mesh.data = src.data
    copy_mesh.data = H5Mesh(group, self._handle)
    for point in src.iter_points():
         copy_mesh.add_point(point)

    ...
    ...
    self.meshes[src.name] = copy_mesh
    return copy_mesh
```